src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,53 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
```

Comment:
just tweak the name a bit and add a space between static and non-static fields
```suggestion
    private static final int MAX_SIZE = 10;
    
```